src/server/services/transfers/TransfersService.cpp

```diff
@@ -58,6 +58,9 @@ TransfersService::TransfersService(): BaseService("TransfersService")
     monitoringMessages = config::ServerConfig::instance().get<bool>("MonitoringMessaging");
     schedulingInterval = config::ServerConfig::instance().get<boost::posix_time::time_duration>("SchedulingInterval");
 
+
+    schedulerFunction = Scheduler::getSchedulerFunction();
+
     Allocator::AllocatorFunction allocatorAlgorithm = Allocator::getAllocatorFunction();
```

Comment:
nit: schedulerFunction and allocatorAlgorithm should probably be named the same way - I vote change allocatorAlgorithm to allocatorFunction